### DISCUSSION TITLE
QT Application using AWS SDK throws redefinitions of function error during Build

### DISCUSSION THREAD

**Discussion by brittosajii** [aws/aws-sdk-cpp#3062] [Q&A] [answered]
![Screenshot 2024-07-31 215639](https://github.com/user-attachments/assets/ff9b916d-7a68-4df5-a446-9c0dd6c353b8)

QT Application using aws-sdk for s3 module throws an showing redefinition of functions in math.gcc-builtin.ini and math.msc.ini
See Screenshot for more information.

Any advice on what I'm doing wrong here will be helpful

**Comment by jmklix** [comment]
Can you please provide more details about this build error?
- sdk version
- OS
- build steps to reproduce
- any other information relevant to this error

**Comment by DmitriyMusatkin** [comment, marked as answer]
Note: its trying to pick up math implementations for both gcc and msvc. So something seems to be off in the build process.

**Reply by jmklix** [reply]
@brittosajii can you make sure you have a clean build environment when building this sdk?
